force-app/main/default/classes/DMLOperations.cls

```diff
@@ -61,14 +64,15 @@ public with sharing class DMLOperations {
      */
     public static Id insertNewContact(Id accountId) {
         // Create a new Contact instance
-
         // Set some fields on the Contact
         // Set the AccountId field to the given accountId
+        Contact contact = new Contact(LastName='Test Contact', AccountId=accountId);
```

Comment:
Minor formatting note, I would put spaces between operators for readability, and then format them vertically to be more readable. As soon as you hit 3+ fields you are assigning on construction, it can get kind of far-right.
```suggestion
        Contact contact = new Contact(
                LastName = 'Test Contact',
                AccountId = accountId
        );
```